tasks/cephfs/test_pool_perm.py

```diff
@@ -80,6 +80,7 @@ def test_forbidden_modification(self):
 
         self.mount_a.umount_wait()
         self.mount_a.mount()
+        self.mount_a.wait_until_mounted()
```

Comment:
So I guess we can be more careful and start using the 'mounted' context manager more widely across the code base and not worry too much about races and if we forgot to umount?
So just,
with self.mount_a.mounted():
as done in 'test_journal_migration.py'